fix for incompatibility between recent Intel MKL versions and numpy
see https://github.com/numpy/numpy/issues/8529
--- molmod/transformations.py.orig	2017-06-28 11:04:49.405580197 +0200
+++ molmod/transformations.py	2017-06-28 11:04:12.324857787 +0200
@@ -469,8 +469,9 @@
     W[0] = 1
     W[1] = 1
     W[2] = 1
-    if numpy.linalg.det(A) < 0:
-        W[2] = -1
+    with numpy.errstate(invalid='ignore'):
+        if numpy.linalg.det(A) < 0:
+            W[2] = -1
     r = numpy.dot(Vt.transpose()*W, U.transpose())
     return Complete(r, numpy.dot(r, -mb) + ma)
 
